### PR TITLE
Move channel parameters to their own conf section

### PR DESCRIPTION
**Pull request by pm47**
For consistency with existing sections: `router`, `relay`, `peer-connection`, etc.

### CODE REVIEW

File: eclair-core/src/test/scala/fr/acinq/eclair/integration/ChannelIntegrationSpec.scala [hunk 1]
```diff
@@ -452,9 +452,9 @@ abstract class ChannelIntegrationSpec extends IntegrationSpec {
...
-    instantiateEclairNode("C", ConfigFactory.parseMap(Map("eclair.node-alias" -> "C", "eclair.expiry-delta-blocks" -> 40, "eclair.fulfill-safety-before-timeout-blocks" -> 12, "eclair.server.port" -> 29741, "eclair.api.port" -> 28091).asJava).withFallback(withAnchorOutputs).withFallback(commonConfig))
-    instantiateEclairNode("F", ConfigFactory.parseMap(Map("eclair.node-alias" -> "F", "eclair.expiry-delta-blocks" -> 40, "eclair.fulfill-safety-before-timeout-blocks" -> 12, "eclair.server.port" -> 29742, "eclair.api.port" -> 28092).asJava).withFallback(withDefaultCommitment).withFallback(commonConfig))
+    instantiateEclairNode("A", ConfigFactory.parseMap(Map("eclair.node-alias" -> "A", "eclair.channel.expiry-delta-blocks" -> 40, "eclair.channel.fulfill-safety-before-timeout-blocks" -> 12, "eclair.server.port" -> 29740, "eclair.api.port" -> 28090).asJava).withFallback(withDefaultCommitment).withFallback(commonConfig))
```
Comment (t-bast) [inline eclair-core/src/test/scala/fr/acinq/eclair/integration/ChannelIntegrationSpec.scala:455]: You forgot to update the corresponding values in the `AnchorOutputChannelIntegrationSpec` and `AnchorOutputZeroFeeHtlcTxsChannelIntegrationSpec` sections below.

Comment (pm47) [reply eclair-core/src/test/scala/fr/acinq/eclair/integration/ChannelIntegrationSpec.scala:455]: Fixed with 25c76a7.

File: eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala [hunk 2]
```diff
@@ -62,36 +62,17 @@ case class NodeParams(nodeKeyManager: NodeKeyManager,
...
-                      maxReserveToFundingRatio: Double,
-                      unhandledExceptionStrategy: UnhandledExceptionStrategy,
                       db: Databases,
-                      revocationTimeout: FiniteDuration,
                       autoReconnect: Boolean,
                       initialRandomReconnectDelay: FiniteDuration,
                       maxReconnectInterval: FiniteDuration,
                       chainHash: ByteVector32,
-                      channelFlags: ChannelFlags,
                       watchSpentWindow: FiniteDuration,
```
Comment (t-bast) [inline eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala:73]: Shouldn't we move this one to `RouterConf`?

Comment (pm47) [reply eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala]: Done with 827eda5.

**Review by t-bast — APPROVED** [review]

**Pull request merged**